app.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "Kutt",
+    "description": "Free Modern URL Shortener.",
+    "website": "https://kutt.it",
+    "repository": "https://github.com/thedevs-network/kutt",
+    "logo": "https://kutt.it/images/logo.svg",
+    "success_url": "/",
+    "env": {
+      "NPM_CONFIG_PRODUCTION": {
+        "value": "false"
+      },
```

Comment:
This is needed for knex migrate to run properly.

https://stackoverflow.com/a/64679533/7168030